catkin_tools_document/builders.py

```diff
@@ -69,7 +69,7 @@ def sphinx(conf, package, deps, output_path, source_path, docs_build_path):
         rpp.insert(0, os.path.join(source_path, 'src'))
     env = {
         'PATH': os.environ['PATH'],
-        'PYTHONPATH': ':'.join(sys.path),
+        'PYTHONPATH': os.environ['PYTHONPATH'],
```

Comment:
`sys.path` has a lot of stuff in it that doesn't come from the PYTHONPATH envvar, include stuff that's specific to the particular python version. This caused me problems because my sphinx executable was still a Python 2 version.